### ISSUE TITLE
Data?

### ISSUE THREAD

**Issue by wangshun1121** [kav019/WGD_Aneuploidy#1]
I cloned your code and tried to run. In R codes, there are several data files, and I went to Cancer Cell 2018 paper's webpage, and got the supplementary data. But there are several data table I can't certain. 

Would you please add a README file and show the link of input data?

**Comment by kav019** [comment]
Yes, doing that is on my to-do list, but I haven't been able to get to it yet because I've been pretty swamped. In the meanwhile if you have questions about specific data, I'll do my best to answer.

**Comment by wangshun1121** [comment]
Well, here are the input files I want:

https://github.com/kav019/WGD_Aneuploidy/blob/583d522b38cd514d9c09dc07cbfaf5805c014d79/Single-event%20analyses/1a%20-%20Aneuploidy%20scores.R#L5

https://github.com/kav019/WGD_Aneuploidy/blob/583d522b38cd514d9c09dc07cbfaf5805c014d79/Single-event%20analyses/1b%20-%20Loss%20vs%20gain%20fraction.R#L4

https://github.com/kav019/WGD_Aneuploidy/blob/583d522b38cd514d9c09dc07cbfaf5805c014d79/Single-event%20analyses/1c%20-%20Fraction%20of%20top%203%20events.R#L5

https://github.com/kav019/WGD_Aneuploidy/blob/583d522b38cd514d9c09dc07cbfaf5805c014d79/Single-event%20analyses/1d%20-%20KL%20div.R#L4

https://github.com/kav019/WGD_Aneuploidy/blob/583d522b38cd514d9c09dc07cbfaf5805c014d79/Single-event%20analyses/1d%20-%20L2%20norm.R#L4

https://github.com/kav019/WGD_Aneuploidy/blob/583d522b38cd514d9c09dc07cbfaf5805c014d79/Single-event%20analyses/1e%20-%20relative_gain_loss.py#L10-L11

https://github.com/kav019/WGD_Aneuploidy/blob/583d522b38cd514d9c09dc07cbfaf5805c014d79/Single-event%20analyses/1f%20-%20Whole%20chr%20anp%20fraction.R#L5

https://github.com/kav019/WGD_Aneuploidy/blob/583d522b38cd514d9c09dc07cbfaf5805c014d79/Single-event%20analyses/3a%20-%20CCLE%20aneuploidy%20scores%20(whole-chr%2Barm).R#L9

https://github.com/kav019/WGD_Aneuploidy/blob/583d522b38cd514d9c09dc07cbfaf5805c014d79/Single-event%20analyses/3b%20-%20CCLE%20whole%20chr%20aneuploidy%20frac.R#L8

https://github.com/kav019/WGD_Aneuploidy/blob/583d522b38cd514d9c09dc07cbfaf5805c014d79/Single-event%20analyses/3d%20-%20CCLE%20KL%20div.R#L4
